contracts/test/LibLiquidationMock.sol

```diff
@@ -72,4 +76,32 @@ library LibLiquidationMock {
             minimalReceipt
         );
     }
+
+    /**
+     * @notice call LibLiquidation.partialLiquidationIsValid
+     */
+    function partialLiquidationIsValid(
```

Comment:
Extremely good candidate for `Balances.Position`.

Reply:
_Somewhat_ hacky in that it's not strictly a delta type, but you can still use this type for the _change in_ parameters, too.

Reply:
I've so far been unable to get this to work when calling external library functions (i.e. in these mock contracts). It always just throws with a some library-related error that is impossible to decypher and only works if you pass in individual values and wrap it in the struct inside the contract function.